src/components/BitcoinBarChart/index.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios'
+import React from 'react'
+import { Bar } from 'react-chartjs-2'
+
+const options = {
+  plugins: {
+    legend: {
+      display: false,
+    },
+  },
+  scales: {
+    y: {
+      grid: {
+        display: false,
+        drawTicks: false,
+        borderWidth: 0,
+      },
+      ticks: {
+        display: false,
+      },
+    },
+    x: {
+      grid: {
+        display: false,
+        borderWidth: 0,
+      },
+      ticks: {},
+    },
+  },
+}
+
+class BitcoinBarChart extends React.Component {
+  state = {
+    dataPoints: [],
+    dataLabels: [],
+  }
+
+  getVolumes = async () => {
+    try {
+      const { data } = await axios(
+        `https://api.coingecko.com/api/v3/coins/bitcoin/market_chart?vs_currency=${this.props.currency}&days=22&interval=daily`,
+      )
+      const volumes = data.total_volumes
+      const dataPoints = []
+      const dataLabels = []
+      volumes.forEach((volume) => {

```

Comment:
use map instead of forEach for transforming data, let me know if you don't know how to do it

Reply:
Done. Used Reduce